nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/WithFrameworkProperty.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  Contributors:
+ *      Kevin Leturc <kleturc@nuxeo.com>
+ */
+
+package org.nuxeo.runtime.test.runner;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.nuxeo.runtime.api.Framework;
+
+/**
+ * Annotation that will help you set {@link Framework}'s {@code property} in your test by annotating class or method.
```

Comment:
i think:  will help you **to** set **the** 

-  your test by annotating class or method means we can add this annotation on class but we have only `@Target(ElementType.METHOD)` on this annotation no the `Type` target